src/config/2020.json

```
@@ -212,6 +212,17 @@
       "website": "https://aagar.github.io/",
       "github": "AAgar",
       "twitter": "AAgar"
+    },
+     "adityapandey1998": {
+      "name": "Aditya Pandey",
+      "teams": [
+       "analysts",
+        "developers"
+      ],
+      "avatar_url": "https://avatars1.githubusercontent.com/u/31750960?v=4&s=200",
+      "github": "adityapandey1998",
+      "twitter": "adityapandey98",
+      "website": "https://www.linkedin.com/in/adityapandey98/"
```

Generally not a fan of adding in LinkedIn as a website (though I note a couple of others have this too). If we want that probably should add a separate social media icon rather than put it under the website.

What do you think @rviscomi ?

```suggestion
      "twitter": "adityapandey98"
```

---

+1 let's add it as a separate social channel for now and we can open a new issue for Contributor page support
```suggestion
      "twitter": "adityapandey98",
      "linkedin": "https://www.linkedin.com/in/adityapandey98/"
```

---

We can merge this and handle the `linkedin` specific changes in #1136.